swagger2.0/oic.sec.doxm.swagger.json

```diff
@@ -88,6 +88,11 @@
           "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
           "type": "string"
         },
+        "timer": {
```

Comment:
Why is this "timer" in the SDI pull request?

Reply:
Wrong base revision, will rebase

Reply:
See https://github.com/openconnectivityfoundation/security/pull/125

Reply:
PR #125 looks good.  We need to be sure to remove/cancel this one so that the wrong one doesn't get accepted.

Reply:
Done